packages/api/src/shared/config.ts

```diff
@@ -277,6 +277,10 @@ export class Config {
     return getEnvVarOrFail("DOCUMENT_DOWNLOADER_LAMBDA_NAME");
   }
 
+  static getFHIRtoBundleLambdaName(): string | undefined {
```

Comment:
Why we wouldnt we fail? We throw an error anyways when calling this

Reply:
done